.circleci/config.yml

```diff
@@ -1,7 +1,73 @@
 version: 2.1
+
 orbs:
   node: circleci/node@3.0.0
+  aws-cli: circleci/aws-cli@1.3.0
+  aws-s3: circleci/aws-s3@1.0.16
+
+parameters:
+  s3-bucket:
+    type: string
+    default: book-a-desk-ui
+
+jobs:
+  build-assets:
+    steps:
+      - checkout
+      - run:
+          name: NPM Build Project
+          command: |
+            npm run build
+      - persist_to_workspace:
+          root: ./dist
+          paths:
+            - ./*
```

Comment:
@TomasMorton I kind of assumed that `root: ./dist` and `paths: ./*` together meant persist everything in `dist/` but don't move it?

It's the same thing we do in platform shell: https://github.com/ayudasystems/platform-shell/blob/master/.circleci/config.yml#L90-L93

Reply:
OK cool :) when you attach, you can attach directly to ./dist if you want right? Not saying to do it, just checking.